app/assets/javascripts/spree/backend/spree-select2.js

```diff
@@ -17,6 +17,12 @@ document.addEventListener("spree:load", function() {
     placeholder: Spree.translations.select_an_option,
     allowClear: true
   })
+
+  if(jQuery().jquery === '3.6.0') {
```

Comment:
Could you describe in more detail what changed in jquery 3.6 so that this `if` is required?
Also, if applying such a change it would be great to do a broader check e.g. if version is `>= 3.6`

Reply:
Hi @rafalcymerys,

This pull request includes a fix for the search box not focusing immediately after the select is opened for clients using latest jquery v3.6.0. Please see the video below for the issue.

[Spree-Administration-Products.webm](https://user-images.githubusercontent.com/13773/187089428-6ede53c2-ea71-464d-b523-43a49338d369.webm)

Note : I have changed the jQuery version check with broader check.